src/main/java/mate.academy.internetshop/dao/Storage.java

```diff
@@ -28,11 +28,11 @@
         Item herbs = new Item("Herbs", 200D);
         items.add(herbs);
 
-        User user = new User();
+        /*User user = new User();
```

Comment:
your repository shouldn't consist commented code

Reply:
Sorry, forgot to remove it. Fixed